gridstatus/pjm.py

```diff
@@ -251,29 +259,119 @@ def get_lmp(
         """
         market = Markets(market)
 
+        if locations == "hubs":
+            locations = self.hub_node_ids
+
+        if location_type:
+            location_type = location_type.upper()
+            if location_type not in self.location_types:
+                raise ValueError(
+                    f"location_type must be one of {self.location_types}",
+                )
+
         if date == "latest":
-            """Currently only supports DAY_AHEAD_HOURlY"""
+            """Supports DAY_AHEAD_HOURlY, REAL_TIME_5_MIN"""
             return self._latest_lmp_from_today(
                 market=market,
                 locations=locations,
                 location_type=location_type,
                 verbose=verbose,
             )
+        if utils.is_today(date, tz=self.default_timezone):
+            if market not in (
+                Markets.REAL_TIME_5_MIN,
+                Markets.DAY_AHEAD_HOURLY,
+            ):
+                raise NotSupported(
+                    f"{market.value} is not supported for latest/today",
+                )
 
-        if locations == "hubs":
-            locations = self.hub_node_ids
+        # Plan which APIs to use
+        if utils.is_today(date, tz=self.default_timezone):
+            fetch_dv = market in (
```

Comment:
because of the above check on line 280, this will always evaluate to True. I think the logic only has to be around `fetch_json`

Reply:
also, we will eventually merge in #135, so no need for the defensive programming